temporian/core/data/duration.py

```diff
@@ -103,3 +111,61 @@ def is_a_date(value: any) -> bool:
         if value is date_type:
             return True
     return False
+
+
+def duration_abbreviation(

```

Comment:
Would be good to have a minimal test directly ensuring this works as expected

Reply:
We test this implicitly in lag due to the name of the new feature